servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -1552,6 +1552,12 @@ bool LightStorage::reflection_probe_instance_postprocess_step(RID p_instance) {
 	if (rpi->processing_side == 6) {
 		rpi->processing_side = 0;
 		rpi->processing_layer++;
+		if (rpi->processing_layer == atlas->reflections[rpi->atlas_index].data.layers[0].mipmaps.size()) {
+			rpi->rendering = false;
+			rpi->processing_side = 0;
```

Comment:
This is redundant with line 1553. Shouldn't hurt, and might prevent potential bugs if line 1553 is changed, but I think some static analysis tools may complain about it?

Reply:
I'll just remove it then. I copied it to keep it the same as the check above, but it would be better just to remove the redundancy

Reply:
Done!